### ISSUE TITLE
Rootcheck false-positives on Busybox-based OS

### ISSUE THREAD

**Issue by jnasselle** [wazuh/wazuh#15947]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| up to master | rootcheck (wazuh-syscheckd)| Manager- Agent | Any |  GNU/Linux Busybox-based OSes (i.e Alpine Linux) |

## Description

As the user states in #15858, rootcheck is raising the next alert

```
Trojaned version of file '/bin/ls' detected. Signature used: 'bash|^/bin/sh|dev/[^clu]|.tmp/lsfile|duarawkz|/prof|/security|file.h' (Generic).
title: Trojaned version of file detected.
file: /bin/ls
```

This is also happening with `/usr/bin/env` and `/bin/echo` as well

## RCA

The problem resides in Busybox bundle, which contains all common/core utils as a single binary, totally different compared to the widely used `coreutils` package that provides one binary per tool.

Rootcheck will check 'bash|^/bin/sh|dev/[^clu]|.tmp/lsfile|duarawkz|/prof|/security|file.h' against ALL Busybox tools and will trigger false-positives since is checking the whole bundle and not only `/bin/ls`

**NOTE**:  [Alpine Linux will be officially supported on v4.4](https://github.com/wazuh/wazuh-packages/pull/1715), so this behavior should be fixed.

**Comment by jnasselle** [comment]
Closed in favor of #15858

**Comment by jnasselle** [comment]
Closed in favor of #15858
